tardis/util/base.py

```diff
@@ -600,6 +600,8 @@ def is_notebook():
 
 
 if is_notebook():
+    import tqdm.notebook
```

Comment:
Why is this change in this PR?

Reply:
That was a mistake, just a temp fix to fix a problem on my machine. I'll revert the file in a new commit.